### PR TITLE
fix wrong parameter in cmd in readme-sycl.md

### PR DESCRIPTION
**Pull request by arthw**
The build cmd in readme-sycl.md is wrong.
"cmake **--build** .." is incorrect after verify.
Correct it.

### CODE REVIEW

**Review by airMeng — APPROVED** [review]

**Review by abhilash1910 — COMMENTED** [review]
Thanks for fix!

File: README-sycl.md [hunk 1]
```diff
@@ -229,12 +229,11 @@ source /opt/intel/oneapi/setvars.sh
 # Build LLAMA with MKL BLAS acceleration for intel GPU
 mkdir -p build && cd build
 
-# Option 1: Use FP16 for better performance in long-prompt  inference
-cmake --build .. -DLLAMA_SYCL=ON -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx -DLLAMA_SYCL_F16=ON
-# Or without "--build", run "make" next
+# Option 1: Use FP16 for better performance in long-prompt inference
+#cmake .. -DLLAMA_SYCL=ON -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx -DLLAMA_SYCL_F16=ON
```
Comment (limitedAtonement) [inline README-sycl.md:233]: This line should not start with `#`.

**Comment by limitedAtonement** [comment]
When running `cmake --build .. -D...`, I get

```
Unknown argument -DLLAMA_SCL=ON
```

Using the instructions this pull request creates works as expected. (`cmake --build .` not `cmake --build ..` is correct in the pull request.)

File: README-sycl.md (hunk 1 above)
Comment (limitedAtonement) [reply README-sycl.md:233]: I see, excellent observation. That is not a good way to notate that, though. I think the `#` should be removed and add to the comment above something like: `. Not generally recommended`.

Comment (NeoZhangJianyu) [reply README-sycl.md:233]: In fact, this cmd is copied from the example/sycl/build.sh at beginning.
But next editor don't know it and modify it again and again. :)

**Comment by NeoZhangJianyu** [comment]
> When running `cmake --build .. -D...`, I get
> 
> ```
> Unknown argument -DLLAMA_SCL=ON
> ```
> 
> Using the instructions this pull request creates works as expected. (`cmake --build .` not `cmake --build ..` is correct in the pull request.)

This method should work well too.
But this cmd will set and build all tools in llama.cpp. it will take more time.

In latest version, there are two cmake cmds:
```
cmake .. -DLLAMA_SYCL=ON -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx
cmake --build . --config Release -j -v
``
The second cmd allow user choose to build one or all tools.
It will save more time.

So, I think current two cmds are better.

**Pull request merged**